src/keeper.ts

```diff
@@ -264,6 +292,88 @@ class Keeper {
         component: "Indexer",
       });
     });
+
+    // update volume metrics
+    await this.updateVolumeMetrics(deps);
+
+    // update open interest metrics
+    await this.updateOIMetrics(deps);
```

Comment:
Currently we're only listening to `PositionLiquidated` and `PositionModified`. See `getEvents` and `EventsOfInterest`.
The two is statements in this function will to what needed for the event and then return (line `273` and`288` .  So I dont think these two functions ever will be called.

Reply:
the early returns are for the function that's being run in `forEach` for each event, these methods are then run after that loop finishes on the updated index. Also the tests for `updateIndex` check the metrics are updated.

Reply:
You are right indeed.. Sorry about that. 
And just too confirm we still only care about `PositionLiquidated` and `PositionModified`. Actually yes since we're only looking at size. And margin added doesn't affect that